Modules/_winapi.c

```diff
@@ -112,6 +112,15 @@ typedef struct {
     Py_buffer write_buffer;
 } OverlappedObject;
 
+static int
+overlapped_traverse(OverlappedObject *self, visitproc visit, void *arg)
+{
```

Comment:
Can you please add a comment somewhere to explain that tp_clear is not implemented, because it requires to cancel the operation if it's pending and the cancellation is quite complex and can fail: see overlapped_dealloc().